src/drivers/hermes/hermes.c

```diff
@@ -42,6 +42,10 @@ extern int xdma_probe_one(struct pci_dev *pdev, const struct pci_device_id *id);
 extern void xdma_remove_one(struct pci_dev *pdev);
 extern void *xdma_pci_dev_get_priv(void *xpdev);
 extern void *xdma_pci_dev_set_priv(void *xpdev, void *x);
+extern void *xdma_get_xcdev_h2c(void *xpdev);
```

Comment:
Instead of putting code in a commit message, provide a commit with a full working example usage.